src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -52,14 +52,6 @@ void Terminal::Create(til::size viewportSize, til::CoordType scrollbackLines, Re
     auto dispatch = std::make_unique<AdaptDispatch>(*this, &renderer, _renderSettings, _terminalInput);
     auto engine = std::make_unique<OutputStateMachineEngine>(std::move(dispatch));
     _stateMachine = std::make_unique<StateMachine>(std::move(engine));
-
-    // Until we have a true pass-through mode (GH#1173), the decision as to
-    // whether C1 controls are interpreted or not is made at the conhost level.
-    // If they are being filtered out, then we will simply never receive them.
-    // But if they are being accepted by conhost, there's a chance they may get
-    // passed through in some situations, so it's important that our state
-    // machine is always prepared to accept them.
-    _stateMachine->SetParserMode(StateMachine::Mode::AlwaysAcceptC1, true);

```

Comment:
💭 Are we guaranteed to always get the C1 mode right, now?

Reply:
The only difference between what conhost and WT see is the injection of CRs into the output when `DISABLE_NEWLINE_AUTO_RETURN` is reset (plus the injection of WIN32IM and focus modes). As such, `DECAC1` as well as any C1 characters should be seen as-is by WT.